api/src/main/java/com/epam/pipeline/security/saml/SAMLUserDetailsServiceImpl.java

```diff
@@ -61,6 +62,9 @@
     @Value("${saml.user.auto.create: EXPLICIT}")
     private SamlUserRegisterStrategy autoCreateUsers;
 
+    @Value("${saml.user.blocked.attribute: false}")
```

Comment:
Could you add this property to dev `application.properties` file and leave it empty? Also, please check that application still starts correctly.

Reply:
Done